test/jdk/java/foreign/TestLinker.java

```diff
@@ -26,7 +26,8 @@
  * @enablePreview
  * @requires jdk.foreign.linker != "UNSUPPORTED"
  * @modules java.base/jdk.internal.foreign
- * @run testng TestLinker
+ * @run testng/othervm/policy=security.policy
+ *          -Djava.security.manager=default TestLinker
```

Comment:
This test should also be run without a `SecurityManager` installed.

Reply:
Good point